examples/load_generation.rb

```diff
@@ -1,21 +1,19 @@
-process :node1, [:node1, :node2, :node3], 2
-process :node2, [:node1, :node2, :node3], 2
-process :node3, [:node1, :node2, :node3], 2
+process :node1, [:node1, :node2, :node3], 3
+process :node2, [:node1, :node2, :node3], 3
+process :node3, [:node1, :node2, :node3], 3
 
 on :node1 do
-  start_load_generation 10000, 10000, 100
+  start_load_generation 1000, 1000, 30
 end
 
-runs = 0
-while runs == 0
+retry_until_true retries: 100 do
   on :node2 do
     m = metrics
     puts "node2: transactions applied: #{m["ledger.transaction.apply"]["count"]}"
     puts "node2: transactions per sec: #{m["ledger.transaction.apply"]["mean_rate"]}"
-    sleep 5
   end
   on :node1 do
     m = metrics
-    runs = m["loadgen.run.complete"]["count"] rescue 0
+    m["loadgen.run.complete"]["count"] > 0
```

Comment:
This will error out in the case the "count" is nil.  That may be fine given this is just an example file, but you could also do:

```ruby
(m["loadgen.run.complete"]["count"] rescue 0) > 0
```